### ISSUE TITLE
Resolve to 0 if resolver is set but addr isn't

### ISSUE THREAD

**Issue by flyswatter** [ethjs/ethjs-ens#5]
Instead of to owner.

Per https://github.com/MetaMask/metamask-plugin/issues/1427

**Comment by kumavis** [comment]
does `0` imply an error or would it be better to explicitly error. dont want to accidentally send to 0.

**Comment by flyswatter** [comment]
Yeah we'll just throw.  I believe we actually throw when an address resolves to 0 already, but just to be clear, we'll throw.
